### DISCUSSION TITLE
Read bacnet device on the same host

### DISCUSSION THREAD

**Discussion by petrosernivka** [JoelBender/bacpypes#522] [Q&A] [answered]
Hi Joel,

How can I read exist bacnet devise (properties and their values) if I run my python code on the same host where this bacnet device launched?

My bacnet device is running on 172.24.188.73:47808 (mask 23).

Thank you

**Comment by JoelBender** [comment, marked as answer]
Two terminal windows...

A) `python3 -m bacpypes3 --address 172.24.188.73/23:47808 --instance 10001 --name "Thing 1"`
B) `python3 -m bacpypes3 --address 172.24.188.73/23:47809 --instance 10002 --name "Thing 2"`

In the second window `read 172.24.188.73 device,10001 object-name`.  In BACnet terms the two applications are running on two different BACnet networks because broadcasts from one will not be received by the other, but the two applications themselves think they are peers because there is no SNET/SADR and DNET/DADR necessary for unicast PDUs.

**Reply by petrosernivka** [reply]
Thanks a lot Joel. It works
